cmd/notary/main_test.go

```diff
@@ -369,3 +370,152 @@ func TestConfigFileOverridenByCmdLineFlags(t *testing.T) {
 	require.Len(t, m.gotten, 1)
 	require.Equal(t, m.gotten[0], "repo.root")
 }
+
+// the config can specify trust pinning settings for TOFUs, as well as pinned Certs or CA
+func TestConfigFileTrustPinning(t *testing.T) {
+	// Set up server that with a self signed cert
+	var err error
+	// add a handler for getting the root
+	m := &recordingMetaStore{MemStorage: *storage.NewMemStorage()}
+	s := httptest.NewUnstartedServer(setupServerHandler(m))
+	s.TLS, err = tlsconfig.Server(tlsconfig.Options{
+		CertFile:   "../../fixtures/notary-server.crt",
+		KeyFile:    "../../fixtures/notary-server.key",
+		CAFile:     "../../fixtures/root-ca.crt",
+		ClientAuth: tls.RequireAndVerifyClientCert,
+	})
+	require.NoError(t, err)
+	s.StartTLS()
+	defer s.Close()
+
+	commander := &notaryCommander{
+		getRetriever: func() passphrase.Retriever { return passphrase.ConstantRetriever("pass") },
+	}
+
+	tempDir := tempDirWithConfig(t, fmt.Sprintf(`{
+		"remote_server": {
+			"url": "%s"
+		},
+		"trust_pinning": {
+		    "tofu": true
```

Comment:
ah yep, great catch!  This is still passing because `disable_tofu` defaults to false and we still have TOFUs :)